releasenotes/notes/fixes_13858-2dacfc0431c1a6ea.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade_synthesis:
+  - |
+    The ``atomic_evolution`` argument to :class:`.ProductFormula` (and its
```

Comment:
If you want to be exhaustive, you should also include `LieTrotter` as a subclass here.